templates/maas/features.html

```diff
@@ -0,0 +1,526 @@
+          <a href="/maas/docs/how-to-enhance-maas-security#p-9102-use-tls-termination-maas-33"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });"
+            aria-label="Link to install and configure secure sockets layer on MAAS documentation"><abbr
+              title="Secure sockets layer">SSL</abbr></a>
+        </li>
+        <li>
+          <a href="/maas/docs/how-to-manage-networks"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });"
+            aria-label="Link to install and configure spanning tree protocol on MAAS documentation"><abbr
+              title="Spanning tree protocol">STP</abbr></a>
+        </li>
+        <li>
+          <a href="/maas/docs/how-to-manage-networks"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });"
+            aria-label="Link to install and configure network proxy on MAAS documentation">Proxy</a>
+        </li>
+      </ul>
+    {%- endif -%}
+  {%- endcall -%}
+  {% call(slot) vf_equal_heights(
+    title_text="Create and manage custom LXD VMs",
+    image_aspect_ratio_small="auto",
+    image_aspect_ratio_medium="auto",
+    image_aspect_ratio_large="auto",
+    items=[
+      {
+        "title_text": 'Graphical overview of your KVM host’s resources',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/87982747-graphical_overview_1.jpg",
+          description="pod details",
+        ),
+        "description_html": '<p>MAAS shows your KVM pod’s CPU cores and RAM, as well as the free space in storage pools.</p>
+          <p>Manage and visualize overcommit ratios.</p>',
+      },
+      {
+        "title_text": 'Choose storage from multiple pools',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/8c09b579-choose_storage_1.jpg",
+          description="storage details",
+        ),
+        "description_html": '<p>Assign one or more volumes from the default and other available pools.</p>
+          <p>View the total size of your requests and the free space remaining.</p>',
+      },
+      {
+        "title_text": 'Customize a VM’s network interfaces',
+        "image_html": lightbox_image(
+          img="https://assets.ubuntu.com/v1/19d80432-customize_1.jpg",
+          description="network interfaces",
+        ),
+        "description_html": '<p>Create software defined networking as complex as you need.</p>
+          <p>Assign interfaces to spaces for Juju models, browse subnets by fabric and VLAN, or simply type an IP number.</p>',
+      },
+    ]
+  ) %}
+    {% if slot == "description" %}
+      <p>
+        Kernel-based Virtual Machine (KVM) is the core Linux kernel technology that enables hardware virtualization. LXD
```

Comment:
bolded in the copydoc
```suggestion
        <strong>Kernel-based Virtual Machine (KVM)</strong> is the core Linux kernel technology that enables hardware virtualization. LXD
```

Reply:
nonblocking, as it's not bolded in the Figma and Design has already approved. [Raised](https://www.figma.com/design/9xLdZg1mLVRqd9ibbzCeo4?node-id=119-4698&m=dev#1472498820) with @mattea-turic @eliman11 in case they'd like to comment.

Reply:
Hi! We don't tend to apply bold styling word-by-work in paragraphs. PM approved, so all should be good :)